app/src/main/res/values-de/strings.xml

```diff
@@ -207,13 +207,13 @@
     <string name="layout_pending_safe_description">Scanne den QR-Code oder teile deine Addresse mit dem Account oder der Person, die dir Geld schicken möchte.</string>
     <string name="safe_address">Safe-Addresse:</string>
     <string name="type">Typ</string>
-    <string name="replace_browser_extension">Ersetze Browser Extension</string>
+    <string name="replace_browser_extension">Ersetze Browser\nExtension</string>
```

Comment:
Can we check were this is used and if it causes any issues

Reply:
![image](https://user-images.githubusercontent.com/17750984/57302457-7edbe600-70db-11e9-8e6f-f9fcc4db3d27.png)
yes, reverting

Reply:
Also this screen does not really include this string. This will for example break in layout_replace_browser_extension.

Reply:
removed new line from strings that were used in single lined text views